### PR TITLE
Integrate ova/vagrant.box packer build scripts

### PR DESCRIPTION
**Pull request by changev**
**Background**
ova/vagrant.box packer build scripts are developed by two people, there's some incompatibility between the two parts of changes, so some works should be done to  integrated all the works. 

**What's in this PR**
1. update ansible version used in rackhd installation
* Prior version is 1.5 default in ubuntu 14.04, but this version has some incompatibility with new version ansible. 
2. apititude install rackhd
* Because of the particularity  of rackhd.deb, apt-get install can't satisfy the needs of rackhd installation.
so change to spawn/expect to install it.
3. modification of packer templates
* the latest templates has some problem with vagrant.box build, now fixed.

@panpan0000 @RackHD/corecommitters

### CODE REVIEW

File: packer/ansible/roles/rackhd-ci-builds/tasks/main.yml [hunk 1]
```diff
@@ -12,14 +12,52 @@
...
+  args:
+    executable: /usr/bin/expect
   with_items:
     - rackhd
   sudo: yes
   when: (rackhd_version is undefined) or (not rackhd_version)
 
 - name: Install RackHD Packages specific version
-  apt: pkg="{{ item }}" force=yes
+  shell: |
```
Comment (cgx027) [inline packer/ansible/roles/rackhd-ci-builds/tasks/main.yml]: Any possibility that L16-L32 be reused here? They seem the same piece of shecll.

File: packer/ansible/roles/rackhd-release/tasks/main.yml [hunk 2]
```diff
@@ -12,14 +12,50 @@
   sudo: yes
 
 - name: Install RackHD Packages latest
-  apt: pkg="{{ item }}" state=latest force=yes
+  shell: |
```
Comment (cgx027) [inline packer/ansible/roles/rackhd-release/tasks/main.yml]: seems duplicate again.

File: packer/ansible/roles/rackhd-ci-builds/tasks/main.yml (hunk 1 above)
Comment (changev) [inline packer/ansible/roles/rackhd-ci-builds/tasks/main.yml]: Thanks, there's an other way to do this:
* create a ```rackhd_install.exp``` and recieve a parameter ```version```
But I think 
**write shell in ansible yml dedirectly** 
is more clear than 
**scp the file to target vm and then run the shell**

**Comment by cgx027** [comment]
@changev The change on removing the duplication using ```include``` looks fantastic. :+1:

File: packer/scripts/dep.sh [hunk 3]
```diff
@@ -4,7 +4,9 @@
 apt-get -y update
 
 # to support using the ansible provisioner
-apt-get -y install ansible
+apt-get -y install expect aptitude python-dev python-pip python-yaml libffi-dev
+pip install --upgrade setuptools
```
Comment (changev) [inline packer/scripts/dep.sh]: for 16.04

**Comment by panpan0000** [comment]
:+1:

**Comment by changev** [comment]
Plz don't merge. There's changing-requirement!!! Thanks!

**Comment by changev** [comment]
test this please

**Comment by changev** [comment]
@yyscamper  @anhou  This pr can be merge now! Thanks!

**Pull request merged**